src/toolkits/feature_engineering/word_counter.cpp

```diff
@@ -18,9 +18,7 @@ namespace feature_engineering {
  * Ported from original count_words implementation.
  */
 void word_count_delimiters_update(flexible_type f, flex_list delimiter_list, bool to_lower,
-                                           std::unordered_map<flexible_type, size_t>& ret_count) {
-
-
+                           std::unordered_map<flexible_type, float>& ret_count, flex_type_enum run_mode, flexible_type f_type = 0) {
```

Comment:
please also pay attention to your code style. Currently, we encourage google style for CPP change. https://google.github.io/styleguide/cppguide.html#Function_Declarations_and_Definitions